packages/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(PACKAGE "libyang")
 set(CPP_PACKAGE "libyang-cpp")
 set(PYTHON_PACKAGE "python3-yang")
+set(TOOLS_PACKAGE "yang-tools")
```

Comment:
This name seems a bit too generic, maybe `libyang-tools` would be better?

Reply:
I picked the same name as it uses on Debian Packages to keep things consistent

Reply:
I see, would it be a serious problem to rename it and so the name would be different in the next Debian release?

Reply:
Contacted Ondřej Surý (our Debian Expert) for his input on renaming debian (sub-)packages.

Reply:
Updated in Debian by @oerdnj and updated in the PR for RPM packages

Reply:
Great!